packages/terra-cli/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cerner/terra-cli",
+  "version": "0.0.0",
+  "description": "Terra Cli",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/terra-cli"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^10.13.0 || >=11.10.1"
```

Comment:
Do you remember why we start the upper limit at a (seemingly) random version of 11.x.x?

Reply:
probably because of eslint, I imagine we could drop this back in this package.